Kernel/Process.cpp

```diff
@@ -273,6 +273,12 @@ ErrorOr<void> Process::attach_resources(NonnullOwnPtr<Memory::AddressSpace>&& pr
 
     auto weak_ptr = TRY(this->try_make_weak_ptr());
     m_procfs_traits = TRY(ProcessProcFSTraits::try_create({}, move(weak_ptr)));
+
+    // This is not actually explicitly verified by any official documentation,
+    // but it's not listed anywhere as being cleared, and rsync expects it to work like this.
```

Comment:
Should we add a test to cover this behavior so it doesn't get accidentally broken in the future?

Reply:
I don't see how this could end up being accidentally broken, but I can look into adding a test case for this later.

That said, testing that involves fork and exec probably has its own pitfalls.

Reply:
I believe we already have tests that fork and exec, so you could probably just copy one

Reply:
These are possibly the most :yakbrain: tests that I have written yet, but they successfully test.